src/main/resources/resource/Arduino/MrlComm/Msg.cpp

```diff
@@ -806,3 +806,7 @@ void Msg::begin(HardwareSerial& hardwareSerial){
 byte Msg::getMethod(){
 	return ioCmd[0];
 }
+
+long Msg::getLastHeartbeat(){

```

Comment:
I don't think we should have this method here.  you should never ask for the last heartbeat.. the heartbeat should just be published.

Reply:
there was 2 lastHeartbeatUpdate in the code. one in mrlComm.cpp and one in Msg.cpp. The one in Msg.cpp is getting updated, but not the one in mrlComm.

since the onDisconnected method is in mrlComm.cpp, it need to have access to the value that is been updated